CraftTweaker2-API/src/main/java/crafttweaker/api/entity/IEntityDefinition.java

```diff
@@ -54,6 +54,9 @@
     @ZenMethod
     IEntity spawnEntity(IWorld world, IBlockPos pos);
 
+    @ZenMethod
+    IEntityDefinition getEntityDefinitionFromString(String string);
```

Comment:
It is a duplicate of `IGame#getEntity`
https://github.com/CraftTweaker/CraftTweaker/blob/1.12/CraftTweaker2-MC1120-Main/src/main/java/crafttweaker/mc1120/game/MCGame.java#L87